packages/ipfs/docs/FAQ.md

```diff
@@ -88,8 +86,14 @@ const node = await IPFS.create({
   },
   libp2p: {
     modules: {
-      transport: [wstar],
-      peerDiscovery: [wstar.discovery]
```

Comment:
Do we want to remove discovery?

Reply:
From the old instructions you have to `new` up a WebRTCStar instance to get the discovery property, but now that throws an error about providing an upgrader.

Could you make a suggested change to correct the config here please?

Reply:
also PRd libp2p to use the tag in this missing doc: https://github.com/libp2p/js-libp2p/pull/713

Reply:
Is #3092 ready for review/merging? It's been draft for ages.